dist/gh.css

```diff
@@ -0,0 +1,835 @@
+.ds6 {
```

Comment:
This file doesn't seem to be referenced by any browser.json. How does the global header consume it?

Reply:
Correct. It was an unintended generated file by the Gulp scripts. I could delete it manually at the end, or write some gulp script to delete it. Either way, I didn't explicitly try to have Gulp create it.